examples/text-generation/README.md

```diff
@@ -356,6 +356,18 @@ QUANT_CONFIG=./quantization_config/maxabs_quant.json python ../gaudi_spawn.py \
 --trim_logits \
 ```
 
+### Running FP8 models on single device
+
+Some bf16 models doesn't fit on one card due to hpu memory limitation, but in fp8 precision they do fit.
+As measurement is being calculated in bf16 precision, to be able to run fp8 model on single card you should use `unify_measurements` script.
+More information on usage of the unifier script can be found in fp8 Habana docs mentioned above.
```

Comment:
paste a link to the read the docs here
https://docs.habana.ai/en/latest/PyTorch/Inference_on_PyTorch/Inference_Using_FP8.html